README.md

```diff
@@ -1 +1,187 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+
+<a name="readme-top"></a>
+
+<!--
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] [OPTIONAL] The image is missing from your repository and hence compromising the aesthetics of your readme. You can retrieve this image from the source of this readme template. 😄